pkgs/tools/misc/ddate/default.nix

```diff
@@ -15,6 +15,6 @@ stdenv.mkDerivation {
     description = "Discordian version of the date program";
     license = stdenv.lib.licenses.publicDomain;
     maintainers = with stdenv.lib.maintainers; [kovirobi];
-    platforms = with stdenv.lib.platforms; linux;
+    platforms = with stdenv.lib.platforms; linux ++ darwin;
```

Comment:
Looking at the `ddate` code this should probably be `platforms.all`

Reply:
Great! I’m mostly ignorant in Nix and didn’t know this was an option.